### PR TITLE
Feature/update auth

### PR DESCRIPTION
**Pull request by ChadRehmKineticData**
Updated authentication for organization and personal repo create handlers.  This was required due to deprecations made by github.

Also fixed bug in the organization repo create handler introduced in previous commit

### CODE REVIEW

File: handlers/github_organization_repo_create_v1/handler/init.rb [hunk 1]
```diff
@@ -37,17 +37,25 @@ def execute()
...
       
       resource = RestClient::Resource.new(url)
 
       data = {name: @parameters['name'], description: @parameters['description']}
 
       data['private'] = true if @parameters['visibility'].downcase == "private"
 
-      puts "Calling URL https://api.github.com/orgs/#{org_name}/repos?access_token=XXX" if @debug_logging_enabled
-      response = resource.post(data.to_json, {content_type: :json})
+      puts "Calling URL https://api.github.com/orgs/#{org_name}/repos" if @debug_logging_enabled
```
Comment (jackboespflug) [inline handlers/github_organization_repo_create_v1/handler/init.rb]: Use the variable here:
`puts "Calling URL #{url}" if @debug_logging_enabled`

File: handlers/github_organization_repo_create_v1/handler/init.rb [hunk 2]
```diff
@@ -37,17 +37,25 @@ def execute()
...
+
       response = nil
```
Comment (jackboespflug) [inline handlers/github_organization_repo_create_v1/handler/init.rb]: This is unnecessary with creating and assigning the variable on the next line.

File: handlers/github_organization_repo_create_v1/handler/init.rb [hunk 3]
```diff
@@ -37,17 +37,25 @@ def execute()
...
+      # create repo
+      response = resource.post(
+        data.to_json, 
+        {
+          content_type: :json,
+          authorization: "token #{@info_values['access_token']}"
+        })
+      puts response
```
Comment (jackboespflug) [inline handlers/github_organization_repo_create_v1/handler/init.rb]: Remove this or use  `if @debug_logging_enabled`

File: handlers/github_personal_repo_create_v1/changelog.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Github_Personal_Repo_Create_v1 (02-29-2016)
+* Initial version.  See README for details.
+
+Github_Repo_Create_Organization_V1 (2021-09-24)
```
Comment (jackboespflug) [inline handlers/github_personal_repo_create_v1/changelog.md]: 1.) The organization repo create changelog info shouldn't go in this file.
2.) You are updating personal repo create handler with authentication changes, so should be noted here.

**Review by jackboespflug — CHANGES_REQUESTED** [review]
Chad, needs a few minor changes.

File: handlers/github_organization_repo_create_v1/handler/init.rb (hunk 3 above)
Comment (ChadRehmKineticData) [reply handlers/github_organization_repo_create_v1/handler/init.rb]: Sorry, leftover from testing.  I'll remove it.

**Review by jackboespflug — APPROVED** [review]

**Pull request merged**